pkg/services/govmomi/service.go

```diff
@@ -394,35 +394,47 @@ func (vms *VMService) reconcileStoragePolicy(ctx context.Context, virtualMachine
 	if err != nil {
 		return errors.Wrap(err, "unable to retrieve storage profile ID")
 	}
-	entities, err := pbmClient.QueryAssociatedEntity(ctx, pbmTypes.PbmProfileId{UniqueId: storageProfileID}, "virtualDiskId")
-	if err != nil {
-		return err
-	}
 
 	var changes []types.BaseVirtualDeviceConfigSpec
 	devices, err := virtualMachineCtx.Obj.Device(ctx)
 	if err != nil {
 		return err
 	}
 
+	arrayOfEntities := make([]pbmTypes.PbmServerObjectRef, 0)
```

Comment:
```suggestion
	diskRefs := make([]pbmTypes.PbmServerObjectRef, 0)
```
If I see correctly

Reply:
Changed :) I didn't commited your suggestion because there are other parts to change here